R/admiral_options.R

```diff
@@ -0,0 +1,194 @@
+admiral_default <- new.env(parent = emptyenv())
```

Comment:
I was running into an "cannot change value of locked binding" errors without it, took some inspiration from [R Packages (2e) Section 8.5](https://r-pkgs.org/Data.html#sec-data-state)

Reply:
I see.

Should we name it more general, e.g., `admiral_env`? Then we can use it for all admiral objects we want to modify.

Reply:
I like this idea, as for other admiral objects we want to modify, do you think it would be a good idea to put this `admiral_env` code-chunk in a "R/aaa.R" script then as well? There may have been some unintended luck involved in naming it "admiral_options.R". In the scenario had I named this script "options_for_admiral.R" instead, scripts that come before alphabetically, wouldn't be able to call `admiral_env` correct?

Reply:
Yes, that's correct.

For now I would leave the code-chunk in `admiral_options.R`. If at some point in the future we need to deviate from the alphabetic order, I would rather use the `@include` tag than names like `aaa.R`.